### PR TITLE
JAMES-2292 Provide documentation for WebAdmin MailQueue endpoints

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by aduprat** [comment]
👍

File: src/site/markdown/server/manage-webadmin.md [hunk 1]
```diff
@@ -591,6 +591,115 @@ Response codes:
  - 400: Group structure or member is not valid
  - 500: Internal error
 
+## Administrating mail queues
```
Comment (mbaechler) [inline src/site/markdown/server/manage-webadmin.md]: s/Administrating/Operating

File: src/site/markdown/server/manage-webadmin.md [hunk 2]
```diff
@@ -591,6 +591,115 @@ Response codes:
...
+
+ - 204: Success (No content)
+ - 400: Invalid request
+ - 404: The mail queue does not exist
+ - 500: Internal error
+
+### Flushing mails from a mail queue
+
+```
+curl -XPATCH http://ip:port/mailQueues/mailQueueName?delayed=true
```
Comment (mbaechler) [inline src/site/markdown/server/manage-webadmin.md]: you are missing the payload : `-d {"delayed": false}`

File: src/site/markdown/server/manage-webadmin.md (hunk 1 above)
Comment (chibenwa) [reply src/site/markdown/server/manage-webadmin.md]: See https://github.com/linagora/james-project/pull/1262#discussion_r164627463

I don't really see the difference between both terms. Especially "Administrating" is used throughout the **WebAdmin** documentation page.

Any rational for such changes?

**Comment by aduprat** [comment]
👍

File: src/site/markdown/server/manage-webadmin.md [hunk 3]
```diff
@@ -685,14 +685,16 @@ Response codes:
...
 
 ```
-curl -XPATCH http://ip:port/mailQueues/mailQueueName?delayed=true
+curl -XPATCH http://ip:port/mailQueues/mailQueueName?delayed=true \
+  -d '{"delayed": false}'
 ```
 
-This request should have the query parameter *delayed* set to *true*.
+This request should have the query parameter *delayed* set to *true*, in order to indicate only delayed mails are affected.
+The payload should set the `delayed` field to false inorder to remove the delay. This is the only supported combinaison,
```
Comment (rouazana) [inline src/site/markdown/server/manage-webadmin.md]: combination

**Comment by aduprat** [comment]
👍

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**